CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v5.8.0 (Thu Apr 02 2021)
+
+#### :rocket: New Feature
+
+- `tldts`
+  - Update cli to process each line of STDIN if no agruments are passed [#1009](https://github.com/remusao/tldts/pull/1009) ([@fulldecent](https://github.com/fulldecent))
+
+#### Authors: 1
+
+- William Entriken ([@fulldecent](https://github.com/fulldecent))
+
+---
+
```

Comment:
Removing this as the CHANGELOG is automatically populated on each release based on PRs.
```suggestion
```